src/main/java/com/mraof/minestuck/item/weapon/PogoWeaponItem.java

```diff
@@ -36,8 +38,7 @@ public boolean hitEntity(ItemStack stack, LivingEntity target, LivingEntity play
 	
 	private double getPogoMotion(ItemStack stack)
 	{
-//		return 0.5 + EnchantmentHelper.getEnchantmentLevel(Enchantment.efficiency.effectId, stack)*0.1;
-		return pogoMotion;
+		return 0.5 + EnchantmentHelper.getEnchantmentLevel(Enchantments.EFFICIENCY, stack)*0.1;
```

Comment:
This loses the use of `pogoMotion`. Instead of reinstating the old equation with efficiency, I'd suggest creating a new equation that combines the efficiency level with the pogoMotion factor.